src/components/duet-date-picker/duet-date-picker.tsx

```diff
@@ -518,13 +518,18 @@ export class DuetDatePicker implements ComponentInterface {
     const nextMonthDisabled =
       maxDate != null && maxDate.getMonth() === focusedMonth && maxDate.getFullYear() === focusedYear
 
-    let minYear = selectedYear - 10
-    let maxYear = selectedYear + 10
+    let minYear = selectedYear
+    let maxYear = selectedYear
     if (minDate) {
-      minYear = Math.max(minYear, minDate.getFullYear())
+      minYear = Math.min(minYear, minDate.getFullYear())
+    } else {
+      minYear -= 10
```

Comment:
If I'm reading it right, can we simplify this logic:

```js
const minYear = minDate ? minDate.getFullYear() : selectedYear - 10
const maxYear = maxDate ? maxDate.getFullYear() : selectedYear + 10
```

My thinking is if there is a min/max date set, we should always use that and ignore the selected year.

Reply:
Sounds good to me.